Justfile

```
@@ -68,6 +68,9 @@ copy-assets:
 generate-vcluster-images version="0.0.0":
   go run -mod vendor ./hack/assets/main.go {{ version }} > ./release/vcluster-images.txt
 
+generate-cli-docs version="0.0.0":
```

I don't think we need the version argument here.

```suggestion
generate-cli-docs:
```